packages/ui/product/Product.tsx

```diff
@@ -0,0 +1,46 @@
+import { CardWrapper, CardDetails, CardComponent } from './styles';
+import StarSVG from './StarSVG';
+import { Title, Card, Image, Text, Button, Rating } from '@mantine/core';
+interface ProductProps {
+  ratingValue?: number;
+}
+
+export function Product({ ratingValue = 0 }: ProductProps) {
+  return (
+    <CardWrapper>
+      <Card shadow="sm" padding="lg" radius="md" withBorder>
+        <Card.Section>
+          <Image
+            src="https://placehold.co/600x500.png?text=Product+Image"
+            alt="Product"
+          />
+        </Card.Section>
+        <CardDetails>
+          <CardComponent className="product-name">
+            <Title order={4} weight={500}>
+              Spicy Korean Chicken BBQ
+            </Title>
+          </CardComponent>
+          <CardComponent className="product-rating">
+            <Rating
+              emptySymbol={<StarSVG fill="#CCCCCC" />}
+              fullSymbol={<StarSVG fill="#FFD500" />}
```

Comment:
Upon re-checking no need to make component for star, convert it to.

Disregard my previous comment. Thanks Bren~

``` tsx
import { ReactComponent as StarSVG} from './star.svg';

<div>
   <StarSVG />
 </div>
```

Reply:
StarSVG removed